models/database.go

```diff
@@ -226,6 +226,11 @@ var databaseSchema = [][]string{
 			ADD COLUMN aws_access_key VARCHAR,
 			ADD COLUMN aws_secret_key VARCHAR`,
 	},
+
+	10: {
+		`ALTER TABLE settings
+			ADD COLUMN alert_manager_address VARCHAR`,
```

Comment:
Nope. We store all settings in a single `jsonb` column. We don't need that transaction.

Reply:
Done in latest commit